opencensus/proto/stats/metrics/metrics.proto

```diff
@@ -235,8 +235,13 @@ message DistributionValue {
     // The context of a span to be attached to Exemplar.
     message SpanContext {
 
-      // The resource name of the span.
-      string span_name = 1;
+      // A unique identifier for a trace showing this exemplar.
+      // It is a 32-character hexadecimal encoding of a 16-byte array.
```

Comment:
Do you think this should be a binary (not hex) encoding in order to match OpenCensus's trace.proto?

The Stackdriver exporter will have to format it into the `projects/%s/traces/%s/etc` format like you said.

Reply:
Yeah I think it makes more sense to avoid mentioning hex encoding in this proto. Updated the description here.

And yes, Stackdriver exporter needs to take care of the encoding and formatting (and only the Stackdriver exporter knows the project id to use).